content/en/dashboards/guide/getting_started_with_wildcard_widget.md

```diff
@@ -0,0 +1,125 @@
+
+1. Navigate to an existing dashboard with a useful widget (Top list of database queries)
+2. Use the widget menu or use the keyboard shortcut (<kbd>Ctrl</kbd>/<kbd>CMD</kbd> + <kbd>C</kbd>) to copy the widget.
+3. In a new dashboard, add a Wildcard widget.
+4. In the editor, clear the default query ({{< img src="/icons/cancel-circle.png" alt="X icon to clear the query" inline="true" width="40px">}}).
+5. Paste the copied request with <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>V</kbd>. The query and associated fields carry over automatically.
+
+{{< img src="/dashboards/guide/analyze_p50_vs_p95_latency_with_the_wildcard_widget/import_widget_walkthrough.mp4" alt="Walkthrough of importing a widget query into the Wildcard widget in Datadog" video=true >}}
+
+## Step 2: Refine the query
+
+In the query editor:
+
+1. Expand the **Data Preview** to identify the fields returned from the query.
+2. Next to your query, click **As** to add an alias to your query. This adds clarity, for example, rename `p50:trace.http.request{*} by {service}`→ `p50`
+
+{{< img src="/dashboards/guide/analyze_p50_vs_p95_latency_with_the_wildcard_widget/refine_query_walkthrough.mp4" alt="Walkthrough of refining a query in the Wildcard widget, including renaming fields and adding a P95 formula" video=true >}}
+
+## Step 3: Auto-generate a visualization
+
+At the top of the query editor:
+
+1. Click the **Define Visual** tab.
+2. Press <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> (Mac) or <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> (Windows/Linux) to open the **Command Palette**.
+3. Select **Auto-select chart**.
+
+{{< img src="/dashboards/guide/analyze_p50_vs_p95_latency_with_the_wildcard_widget/command_palette.mp4" alt="Your image description" video=true >}}
+
+Datadog automatically creates a visualization based on your query.
+
+<div class="alert alert-info">
+<strong>Tip:</strong> Use the Command Palette (<kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) to auto-select a chart type based on your query, add or edit encodings, or rotate axes/switch chart types.
+</div>
+
+## Step 4: Add a context menu
+
+To add interactivity to your graph, enable context menu support:
+
+1. In the visual JSON editor, add the following:
+   ```json
+    "params": [
+        {
+            "name": "datadogPointSelection",
+            "select": "point"
+        }
+    ]
+   ```
+2. Save and run the widget.
+3. Click any data point in your wildcard widget to bring up a **context menu** with dynamic filters.
+
+For more information, see [Using Vega-Lite with Wildcard Widgets in Datadog][6].
+
+## Next Steps
+
+You can do any number of customizations with Wildcard widgets:
+- [Change the thickness of lines to show weight and intensity][7]
+- [Add images to represent values][8]
+- [Layer visualizations][9]
```

Comment:
```suggestion
- [Layering visualizations for richer context[9]
```
Suggest changing these to verb phrasing and add a bit more detail but feel free to keep as is !